batch/sql/add-batch-updates.sql

```diff
@@ -0,0 +1,285 @@
+START TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS `batch_updates` (
+  `batch_id` BIGINT NOT NULL,
+  `update_id` INT NOT NULL,
```

Comment:
As far as I know, InnoDB doesn't do auto-increment on the second column of a primary key, as it would require a counter per unique value of `batch_id`. But I would love to be proven wrong here and not have to do the calculation ourselves.

Reply:
It looks like we could implement this using a trigger: https://stackoverflow.com/questions/19786665/mysql-second-auto-increment-field-based-on-foreign-key

Reply:
Since we have to grab the last row to get the previous start_job_id + n_jobs, I think we might as well do the `update_id` calculation in there too.

Reply:
We will still get uniqueness from the primary key.